lib/config/validation.ts

```diff
@@ -205,10 +210,12 @@ export async function validateConfig(
         });
       }
       if (!optionTypes[key]) {
-        errors.push({
-          topic: 'Configuration Error',
-          message: `Invalid configuration option: ${currentPath}`,
-        });
+        if (!earlyCfg.allowExtraConfig) {
+          errors.push({
+            topic: 'Configuration Error',
+            message: `Invalid configuration option: ${currentPath}. Run the server with '--allow-extra-config' flag if you want to allow extra configurations`,
```

Comment:
@HonkingGoose Do we call Renovate "the server" in user facing messages?

Reply:
```suggestion
            message: `Invalid configuration option: ${currentPath}. Run Renovate bot with the '--allow-extra-config' flag if you want to allow extra configurations.`,
```

I think something like this makes more sense.

> @HonkingGoose Do we call Renovate "the server" in user facing messages?

I don't know, you should check the code. But I think it sounds odd when we call it the server... 😉